Content.Shared/Follower/FollowerSystem.cs

```diff
@@ -249,20 +250,36 @@ public void StopFollowingEntity(EntityUid uid, EntityUid target, FollowedCompone
     }
 
     /// <summary>
-    /// Get the most followed entity.
+    /// Gets the entity with the most non-admin ghosts following it.
     /// </summary>
-    public EntityUid? GetMostFollowed()
+    public EntityUid? GetMostGhostFollowed()
     {
+        // Keep a tally of how many ghosts are following each entity
+        var followedEnts = new Dictionary<EntityUid, int>();
+
+        // Look for followers that are ghosts and are player controlled
+        var query = EntityQueryEnumerator<FollowerComponent, GhostComponent, ActorComponent>();
+        while (query.MoveNext(out var _, out var follower, out var ghost, out var _))
+        {
+            // Exclude admin ghosts
+            if (!ghost.CanGhostInteract)
+            {
+                var followed = follower.Following;
+                // Add new entry or increment existing
+                if (!followedEnts.TryAdd(followed, 1))
+                    followedEnts[followed]++;
+            }
+        }
+
+        // Find the entity with the most followers
         EntityUid? picked = null;
-        int most = 0;
-        var query = EntityQueryEnumerator<FollowedComponent>();
-        while (query.MoveNext(out var uid, out var comp))
+        var most = 0;
+        foreach (var followed in followedEnts)
         {
-            var count = comp.Following.Count;
-            if (count > most)
+            if (followed.Value > most)
             {
-                picked = uid;
-                most = count;
+                picked = followed.Key;
+                most = followed.Value;
```

Comment:
This loop can be done in the other one by tracking most there.

Reply:
Good call